tests/test_react_auth_defs.py

```diff
@@ -130,3 +130,13 @@ def mock_foursight_env_name(env):
         return "env-c-foursight-name"
     else:
         return env
+
+def mock_short_env_name(env):
+    if env.startswith("env-a"):
+        return "env-a-short-name"
```

Comment:
Any reason you don't just append `-short-name` to any name that doesn't have it in order to make a short name?

And yes, I got the joke about it being longer. :)

Reply:
If the name is e.g. env-a-full-name then that would end up producing env-a-full-name-short-name which won't match. But you did inspire my to generalize/simplify this, e.g.:
``known_env = [ known_env for known_env in KNOWN_ENVS if env in known_env.values() ]``
``return known_env[0].get("short_name") if len(known_env) == 1 else None``